dirty_cat/tests/test_fuzzy_join.py

```diff
@@ -255,6 +271,111 @@ def fit(self, X):
         fuzzy_join(left, right, on="key", how="left", encoder="awrongencoder")
 
 
+def test_numerical_column():
+    """
+    Testing that fuzzy_join works with numerical columns.
+    """
+
+    left = pd.DataFrame({"str1": ["aa", "a", "bb"], "int": [10, 2, 5]})
+    right = pd.DataFrame(
+        {
+            "str2": ["aa", "bb", "a", "cc", "dd"],
+            "int": [55, 6, 2, 15, 6],
+        }
+    )
+
+    fj_num = fuzzy_join(left, right, on="int", numerical_match="number")
+    n_cols = left.shape[1] + right.shape[1]
+
+    assert fj_num.shape == (len(left), n_cols)
+
+    fj_num2 = fuzzy_join(
+        left, right, on="int", numerical_match="number", return_score=True
+    )
+    assert fj_num2.shape == (len(left), n_cols + 1)
+
+    fj_num3 = fuzzy_join(
+        left,
+        right,
+        on="int",
+        numerical_match="number",
+        match_score=0.9,
+        drop_unmatched=True,
+    )
+    assert fj_num3.shape == (2, n_cols)
+
+    with pytest.raises(
+        ValueError,
+        match=r"The columns you are trying ",
```

Comment:
This match could be improved to include more informative keywords on the error that's to be raised.

Reply:
Thanks @LilianBoulard for the suggestions and improvements.